src/ucp/api/ucp.h

```diff
@@ -1295,6 +1308,75 @@ ucs_status_t ucp_mem_unmap(ucp_context_h context, ucp_mem_h memh);
 
 /**
  * @ingroup UCP_MEM
+ * @brief list of UCP memory use advices.
+ */
+typedef enum ucp_mem_advices {
```

Comment:
the plural of advice is just 'advice'.  So best, I think, to just change 'advices' to 'advice'.